src/test/java/BaseTest/TestRunner.java

```diff
@@ -0,0 +1,21 @@
+package BaseTest;
+
+import Browser.BrowserFactory;
+import Browser.BrowserСhoice;
+import Steps.MainPage;
+import org.testng.annotations.AfterTest;
+
+public class TestRunner {
+    final static String baseUrl = "https://www.citrus.com.ua/uk/";
+
+    public static MainPage loadApplication(){
```

Comment:
Looks like most of your test starts from calling this method. I guess you can mark it with `@BeforeMethod(alwaysRun = true)` annotation to automatically run before each test so you don't need to duplicate same line in every test